user/languages/javascript-with-nodejs.md

```diff
@@ -134,6 +134,20 @@ For more information, refer to [Caching](/user/caching) documentation.
 
 Note that `npm install` can fail if a shrinkwrapped git dependency pointing to a branch has its HEAD changed.
 
+### Using specific version of `npm` or `yarn`
+
+Add the following to the `before_install` phase of `.travis.yml`:
+
+```yaml
+before_install:
+  # for npm
+  - npm i -g npm@version-number
+  
```

Comment:
This YAML snippet should be split into two parts, since running both of them in a build is probably not what the users typically need.